packages/block/src/types.ts

```diff
@@ -71,6 +71,32 @@ export interface BlockOptions {
   skipConsensusFormatValidation?: boolean
 }
 
+export interface VerkleProof {
+  commitmentsByPath: PrefixedHexString[]
+  d: PrefixedHexString
+  depthExtensionPresent: PrefixedHexString
+  ipaProof: {
+    cl: PrefixedHexString[]
+    cr: PrefixedHexString[]
+    finalEvaluation: PrefixedHexString
+  }
+  otherStems: PrefixedHexString[]
+}
+
+export interface VerkleStateDiff {
+  stem: PrefixedHexString
+  suffixDiffs: {
+    currentValue: PrefixedHexString | null
+    newValue: PrefixedHexString | null
+    suffix: number
+  }[]
+}
+
+export interface VerkleExecutionWitness {
+  stateDiff: VerkleStateDiff[]
+  verkleProof: VerkleProof
+}
```

Comment:
Can you please add some general comments here on the datastructure types and also notes that these are experimental?

Reply:
Done